mvs_eland_tool/mvs_eland_tool.py

```diff
@@ -78,7 +79,7 @@ def main(**kwargs):
     if not user_input["path_input_file"].endswith("json"):
         logging.debug("Accessing script: A1_csv_to_json")
         path_to_json_from_csv = load_data_from_csv.create_input_json(
-            input_directory=user_input["path_input_folder"]
+            input_directory=os.path.dirname(user_input["path_input_file"])
```

Comment:
This will not work if I provide path to csv folder then, no?

Reply:
Providing a path to a folder as `path_input_file` would already throw an error in [`get_user_input()`](https://github.com/rl-institut/mvs_eland/blob/4e8e3e2fc663f29b2505d62eb14583d63b790d54/src/A0_initialization.py#L193).

Reply:
`user_input["path_input_folder"]` is the value returned by `check_input_directory(path_input_file)`. It can already be a folder in which your csv folders lie.

`os.path.dirname(user_input["path_input_file"])` does not make sense to me in the case where your input *is* a folder with csv files.

If you provide a path to the csv files as input if throws an error?

The easiest would be for you to write the command you are executing, the output you would expect and the output you currently get (i.e. the backbone of a test), that way I am sure of what you mean ;)